pkg/kapp/config/default.go

```diff
@@ -386,10 +386,18 @@ changeGroupBindings:
   resourceMatchers: &crdMatchers
   - apiGroupKindMatcher: {kind: CustomResourceDefinition, apiGroup: apiextensions.k8s.io}
 
+- name: change-groups.kapp.k14s.io/crds-{crd-group}-{name}
+  resourceMatchers: &crdMatchers
```

Comment:
We can reuse crdMatcher from above instead of redeclaring again.

Reply:
Makes sense, will do.